code/index.html

```diff
@@ -1,22 +1,174 @@
 <!DOCTYPE html>
 <html>
+
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+        <title>Project 2: The Wheeeekly News</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Yusei Magic">
+    <style>
+      body {
+        font-family: 'Yusei Magic', sans-serif;
+      }      
+      </style>
+      <meta name="viewport" content="width=device-width, initial-scale=1">
   </head>
-  <body>
+
+  <body class="container">
+
     <header>
-      <!-- Some kind of logo and navbar -->
-    </header>
+      <div class="left">
```

Comment:
Would be great if these class names had a bit more context, such as `main-container` or `main-content` and `main-container-left` or `main-content-left` 

This way, if you have a `container` on another page that has a `container` you can avoid confusion if it uses the same CSS file